docs/install/browser/README.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Browser
+---
+
+# IPFS browser companion
```

Comment:
"IPFS Companion" is a title. I wouldn't think of looking for it in a section called browser. can we rename this to "IPFS Companion" with children "Chrome Extension" and "Firefox Extension"?

Reply:
06e8275 fixes this issue.